test/suites-ostree/upgrade.robot

```diff
@@ -0,0 +1,85 @@
+*** Settings ***
+Documentation       Tests related to MicroShift upgradeability on OSTree-based systems
```

Comment:
We may want to name this file "healthy-upgrade" because I expect the flow for an unhealthy upgrade to be a bit different.